app/lib/activitypub/activity/create.rb

```diff
@@ -42,62 +46,77 @@ def find_existing_status
   end
 
   def process_status_params
-    {
-      uri: @object['id'],
-      url: object_url || @object['id'],
-      account: @account,
-      text: text_from_content || '',
-      language: detected_language,
-      spoiler_text: text_from_summary || '',
-      created_at: @object['published'],
-      override_timestamps: @options[:override_timestamps],
-      reply: @object['inReplyTo'].present?,
-      sensitive: @object['sensitive'] || false,
-      visibility: visibility_from_audience,
-      thread: replied_to_status,
-      conversation: conversation_from_uri(@object['conversation']),
-      media_attachment_ids: process_attachments.take(4).map(&:id),
-    }
-  end
-
-  def process_tags(status)
+    @params = begin
```

Comment:
why `begin` here? shouldn't this just be `@params = {` ?

Reply:
Otherwise rubocop will require me to align the hash with the `=` which sucks. begin/end does nothing except improve the formatting.

Reply:
which rule was it? `Layout/AlignHash` definitely doesn't work that way. We should make sure to change the rubocop lint formatting when we run into a situation where it's behaving badly, so that we're consistent across PRs

Reply:
As far as I am aware we are required to:

```ruby
hash = {
           foo: bar,
           baz: boo,
         }
```

Which looks bad.

Reply:
My understanding is that the default Layout/AlignHash and [Layout/IndentHash](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/IndentHash) rule should pass with this code:

    hash = {
      foo: bar,
      baz: boo,
    }